### PR TITLE
Fix Asset IDs and resolve xrefs for TODO messages.

### PR DESCRIPTION
**Pull request by tompratt-AQ**
# Title
Fix Asset IDs and resolve xrefs for TODO messages.

## Summary
This resolves issues from PR 1886 which replaced some assetid: tags with TODO messages containing xrefs for entity references that needed to be confirmed and entered correctly in the topics.

PR1886 should now be closed and replaced by this PR.

## Suggested Reviewers
@rpetrusha

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/wcf/add-of-backuplist.md [hunk 1]
```diff
@@ -53,4 +53,4 @@ Represents a configuration element that defines a backup endpoint element.
 |[\<routing>](../../../../../docs/framework/configure-apps/file-schema/wcf/routing.md)|Contains a list of endpoints that you would like the Routing Service to use in case the primary endpoint can't be reached.|  
   
 ## See Also  
- [System.ServiceModel.Routing.Configuration.BackupEndpointElement](assetId:///System.ServiceModel.Routing.Configuration.BackupEndpointElement?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ [<xref:System.ServiceModel.Routing.Configuration.BackupEndpointElement>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/add-of-backuplist.md]: this should include "?displayProperty=fullName" I think that the opening bracket needs to be removed as well.

File: docs/framework/configure-apps/file-schema/wcf/add-of-entries.md [hunk 2]
```diff
@@ -57,5 +57,5 @@ Represents a routing entry that maps a filter to a client endpoint that was prev
 |[\<routing>](../../../../../docs/framework/configure-apps/file-schema/wcf/routing.md)|A configuration section that contains routing mapping entries.|  
   
 ## See Also  
- [System.ServiceModel.Routing.Configuration.RoutingSection](assetId:///System.ServiceModel.Routing.Configuration.RoutingSection?qualifyHint=False&amp;autoUpgrade=True)   
- [System.ServiceModel.Routing.Configuration.FilterTableEntryElement](assetId:///System.ServiceModel.Routing.Configuration.FilterTableEntryElement?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ <xref:System.ServiceModel.Routing.Configuration.RoutingSection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/add-of-entries.md]: Again, both of these should include ?displayProperty=fullName

File: docs/framework/configure-apps/file-schema/wcf/add-of-namespacetable.md [hunk 3]
```diff
@@ -54,4 +54,4 @@ Represents a configuration element that contains a namespace to prefix mapping t
 |[\<namespaceTable>](../../../../../docs/framework/configure-apps/file-schema/wcf/namespacetable.md)|Represents a configuration section for defining a set of elements that contain namespace to prefix mappings that can then be used in XPath filters for routing.|  
   
 ## See Also  
- [System.ServiceModel.Routing.Configuration.NamespaceElement](assetId:///System.ServiceModel.Routing.Configuration.NamespaceElement?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ <xref:System.ServiceModel.Routing.Configuration.NamespaceElement>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/add-of-namespacetable.md]: This should also include "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/add-of-protocolmapping.md [hunk 4]
```diff
@@ -67,5 +67,5 @@ Represents a default protocol mapping between a transport protocol scheme (e.g.,
 ```  
   
 ## See Also  
- [System.ServiceModel.Configuration.ProtocolMappingSection](assetId:///System.ServiceModel.Configuration.ProtocolMappingSection?qualifyHint=False&amp;autoUpgrade=True)   
- [System.ServiceModel.Configuration.ProtocolMappingElement](assetId:///System.ServiceModel.Configuration.ProtocolMappingElement?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ <xref:System.ServiceModel.Configuration.ProtocolMappingSection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/add-of-protocolmapping.md]: Both of these should include  "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/backuplist.md [hunk 5]
```diff
@@ -77,4 +77,4 @@ Represents a configuration section for defining a backup list that enumerates a
 ```  
   
 ## See Also  
- [System.ServiceModel.Routing.Configuration.BackupEndpointCollection](assetId:///System.ServiceModel.Routing.Configuration.BackupEndpointCollection?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ <xref:System.ServiceModel.Routing.Configuration.BackupEndpointCollection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/backuplist.md]: "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/backuplists.md [hunk 6]
```diff
@@ -51,4 +51,4 @@ Represents a configuration section for defining a set of backup services used in
 |[\<routing>](../../../../../docs/framework/configure-apps/file-schema/wcf/routing.md)|Represents a configuration section for defining a set of routing filters, which determine the type of [!INCLUDE[indigo1](../../../../../includes/indigo1-md.md)]<xref:System.ServiceModel.Dispatcher.MessageFilter> to be used when evaluating incoming messages, as well as routing tables that define the target endpoints to send messages to when a filter matches.|  
   
 ## See Also  
- [System.ServiceModel.Routing.Configuration.BackupListCollection](assetId:///System.ServiceModel.Routing.Configuration.BackupListCollection?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ <xref:System.ServiceModel.Routing.Configuration.BackupEndpointCollection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/backuplists.md]: This should include "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/bookmarkresumptionqueries-of-wcf.md [hunk 7]
```diff
@@ -52,7 +52,7 @@ Represents a collection of queries that are used to track resumption of a bookma
 |[\<workflow>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/workflow.md)|A configuration element that contains all queries for a specific workflow identified by the `activityDefinitionId` property.|  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.BookmarkResumptionQueryElementCollection](assetId:///System.ServiceModel.Activities.Tracking.Configuration.BookmarkResumptionQueryElementCollection?qualifyHint=False&amp;autoUpgrade=True)   
- [System.Activities.Tracking.BookmarkResumptionQuery](assetId:///System.Activities.Tracking.BookmarkResumptionQuery?qualifyHint=False&amp;autoUpgrade=True)   
+ <xref:System.ServiceModel.Activities.Tracking.Configuration.BookmarkResumptionQueryElementCollection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/bookmarkresumptionqueries-of-wcf.md]: again,  "?displayProperty=fullName" for both of these

File: docs/framework/configure-apps/file-schema/wcf/bookmarkresumptionquery-of-wcf.md [hunk 8]
```diff
@@ -52,7 +52,7 @@ Represents a query that is used to track resumption of a bookmark within a workf
 |[\<bookmarkResumptionQueries>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/bookmarkresumptionqueries.md)|Represents a collection of queries that are used to track resumption of a bookmark within a workflow instance.|  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.BookmarkResumptionQueryElementCollection](assetId:///System.ServiceModel.Activities.Tracking.Configuration.BookmarkResumptionQueryElementCollection?qualifyHint=False&amp;autoUpgrade=True)   
- [System.Activities.Tracking.BookmarkResumptionQuery](assetId:///System.Activities.Tracking.BookmarkResumptionQuery?qualifyHint=False&amp;autoUpgrade=True)   
+ <xref:System.ServiceModel.Activities.Tracking.Configuration.BookmarkResumptionQueryElementCollection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/bookmarkresumptionquery-of-wcf.md]: "?displayProperty=fullName" for both of these

File: docs/framework/configure-apps/file-schema/wcf/cancelrequestedquery-of-wcf.md [hunk 9]
```diff
@@ -53,7 +53,7 @@ Represents a query that is used to track requests to cancel a child activity by
 |[\<faultPropagationQuery>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/faultpropagationquery.md)|Represents a list of configuration elements that are used to track requests to cancel a child activity by the parent activity. The query is necessary for a tracking participant to subscribe to cancel request record objects.|  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.CancelRequestQueryElement](assetId:///System.ServiceModel.Activities.Tracking.Configuration.CancelRequestQueryElement?qualifyHint=False&amp;autoUpgrade=True)   
- [System.Activities.Tracking.CancelRequestedQuery](assetId:///System.Activities.Tracking.CancelRequestedQuery?qualifyHint=False&amp;autoUpgrade=True)   
+ <xref:System.ServiceModel.Activities.Tracking.Configuration.CancelRequestQueryElement>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/cancelrequestedquery-of-wcf.md]: "?displayProperty=fullName" for both of these.

File: docs/framework/configure-apps/file-schema/wcf/customtrackingqueries-of-wcf.md [hunk 10]
```diff
@@ -51,7 +51,7 @@ Represents a collection of queries that are used to track events that you define
 |[\<workflow>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/workflow.md)|A configuration element that contains all queries for a specific workflow identified by the `activityDefinitionId` property.|  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.CustomTrackingQueryElementCollection](assetId:///System.ServiceModel.Activities.Tracking.Configuration.CustomTrackingQueryElementCollection?qualifyHint=False&amp;autoUpgrade=True)   
- [System.Activities.Tracking.CustomTrackingQuery](assetId:///System.Activities.Tracking.CustomTrackingQuery?qualifyHint=False&amp;autoUpgrade=True)   
+ <xref:System.ServiceModel.Activities.Tracking.Configuration.CustomTrackingQueryElementCollection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/customtrackingqueries-of-wcf.md]: "?displayProperty=fullName" for both.

File: docs/framework/configure-apps/file-schema/wcf/customtrackingquery-of-wcf.md [hunk 11]
```diff
@@ -53,7 +53,7 @@ Represents a collection of queries that are used to track events that you define
 |[\<customTrackingQuery>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/customtrackingquery.md)|A query that is used to track events that you define in your code activities.|  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.CustomTrackingQueryElementCollection](assetId:///System.ServiceModel.Activities.Tracking.Configuration.CustomTrackingQueryElementCollection?qualifyHint=False&amp;autoUpgrade=True)   
- [System.Activities.Tracking.CustomTrackingQuery](assetId:///System.Activities.Tracking.CustomTrackingQuery?qualifyHint=False&amp;autoUpgrade=True)   
+ <xref:System.ServiceModel.Activities.Tracking.Configuration.CustomTrackingQueryElementCollection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/customtrackingquery-of-wcf.md]: "?displayProperty=fullName" for both

File: docs/framework/configure-apps/file-schema/wcf/entries.md [hunk 12]
```diff
@@ -53,5 +53,5 @@ A routing entry that contain mappings between the routing filters and the target
 |[\<routing>](../../../../../docs/framework/configure-apps/file-schema/wcf/routing.md)|A configuration section that contains a routing table.|  
   
 ## See Also  
- [System.ServiceModel.Routing.Configuration.RoutingSection](assetId:///System.ServiceModel.Routing.Configuration.RoutingSection?qualifyHint=False&amp;autoUpgrade=True)   
- [System.ServiceModel.Routing.Configuration.FilterTableEntryElement](assetId:///System.ServiceModel.Routing.Configuration.FilterTableEntryElement?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ <xref:System.ServiceModel.Routing.Configuration.RoutingSection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/entries.md]: "?displayProperty=fullName" for both

File: docs/framework/configure-apps/file-schema/wcf/faultpropagationqueries-of-wcf.md [hunk 13]
```diff
@@ -51,7 +51,7 @@ Represents a collection of queries that are used to track the handling of faults
 |[\<workflow>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/workflow.md)|A configuration element that contains all queries for a specific workflow identified by the `activityDefinitionId` property.|  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.FaultPropagationQueryElementCollection](assetId:///System.ServiceModel.Activities.Tracking.Configuration.FaultPropagationQueryElementCollection?qualifyHint=False&amp;autoUpgrade=True)   
- [System.Activities.Tracking.FaultPropagationQuery](assetId:///System.Activities.Tracking.FaultPropagationQuery?qualifyHint=False&amp;autoUpgrade=True)   
+ <xref:System.ServiceModel.Activities.Tracking.Configuration.FaultPropagationQueryElementCollection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/faultpropagationqueries-of-wcf.md]: "?displayProperty=fullName" for both

File: docs/framework/configure-apps/file-schema/wcf/faultpropagationquery-of-wcf.md [hunk 14]
```diff
@@ -53,7 +53,7 @@ Represents a query that is used to track the handling of faults that occur withi
 |[\<faultPropagationQueries>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/faultpropagationqueries.md)|Represents a list of configuration elements that are used to track the handling of faults that occur within an activity.  This event occurs each time a FaultHandler processes a fault.|  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.FaultPropagationQueryElement](assetId:///System.ServiceModel.Activities.Tracking.Configuration.FaultPropagationQueryElement?qualifyHint=False&amp;autoUpgrade=True)   
- [System.Activities.Tracking.FaultPropagationQuery](assetId:///System.Activities.Tracking.FaultPropagationQuery?qualifyHint=False&amp;autoUpgrade=True)   
+ <xref:System.ServiceModel.Activities.Tracking.Configuration.FaultPropagationQueryElement>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/faultpropagationquery-of-wcf.md]: "?displayProperty=fullName" for both

File: docs/framework/configure-apps/file-schema/wcf/filter.md [hunk 15]
```diff
@@ -63,6 +63,6 @@ None.
 
 ## See also
 
-[System.ServiceModel.Routing.Configuration.FilterElement](assetId:///System.ServiceModel.Routing.Configuration.FilterElement?qualifyHint=False&amp;autoUpgrade=True)
+<xref:System.ServiceModel.Routing.Configuration.FilterElement>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/filter.md]: All three of these should have  "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/filters-of-routing.md [hunk 16]
```diff
@@ -51,4 +51,4 @@ Represents a configuration section for defining a set of routing filters, which
 |[\<routing>](../../../../../docs/framework/configure-apps/file-schema/wcf/routing.md)|Represents a configuration section for defining a set of routing filters, which determine the type of [!INCLUDE[indigo1](../../../../../includes/indigo1-md.md)]<xref:System.ServiceModel.Dispatcher.MessageFilter> to be used when evaluating incoming messages, as well as routing tables that define the target endpoints to send messages to when a filter matches.|  
   
 ## See Also  
- [System.ServiceModel.Routing.Configuration.FilterElement](assetId:///System.ServiceModel.Routing.Configuration.FilterElement?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ <xref:System.ServiceModel.Routing.Configuration.FilterElement>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/filters-of-routing.md]: "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/filtertable.md [hunk 17]
```diff
@@ -55,4 +55,4 @@ Represents a routing table that contains a list of filters to evaluate messages
 |[\<routing>](../../../../../docs/framework/configure-apps/file-schema/wcf/routing.md)|A configuration section that contains routing tables.|  
   
 ## See Also  
- [System.ServiceModel.Routing.Configuration.RoutingSection](assetId:///System.ServiceModel.Routing.Configuration.RoutingSection?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ <xref:System.ServiceModel.Routing.Configuration.RoutingSection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/filtertable.md]: "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/filtertables.md [hunk 18]
```diff
@@ -51,5 +51,5 @@ Represents a configuration section for defining routing tables that contain mapp
 |[\<routing>](../../../../../docs/framework/configure-apps/file-schema/wcf/routing.md)|A configuration section that contains routing filters and routing tables.|  
   
 ## See Also  
- [System.ServiceModel.Routing.Configuration.RoutingSection](assetId:///System.ServiceModel.Routing.Configuration.RoutingSection?qualifyHint=False&amp;autoUpgrade=True)   
- [System.ServiceModel.Routing.Configuration.FilterTableCollection](assetId:///System.ServiceModel.Routing.Configuration.FilterTableCollection?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ <xref:System.ServiceModel.Routing.Configuration.RoutingSection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/filtertables.md]: "?displayProperty=fullName" for both

File: docs/framework/configure-apps/file-schema/wcf/namespacetable.md [hunk 19]
```diff
@@ -52,4 +52,4 @@ Represents a configuration section for defining a set of elements that contain n
 |[\<routing>](../../../../../docs/framework/configure-apps/file-schema/wcf/routing.md)|Represents a configuration section for defining a set of routing filters, which determine the type of [!INCLUDE[indigo1](../../../../../includes/indigo1-md.md)]<xref:System.ServiceModel.Dispatcher.MessageFilter> to be used when evaluating incoming messages, as well as routing tables that define the target endpoints to send messages to when a filter matches.|  
   
 ## See Also  
- [System.ServiceModel.Routing.Configuration.NamespaceElementCollection](assetId:///System.ServiceModel.Routing.Configuration.NamespaceElementCollection?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ <xref:System.ServiceModel.Routing.Configuration.NamespaceElementCollection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/namespacetable.md]: "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/protocolmapping.md [hunk 20]
```diff
@@ -64,5 +64,5 @@ Represents a configuration section for defining a set of default protocol mappin
 ```  
   
 ## See Also  
- [System.ServiceModel.Configuration.ProtocolMappingSection](assetId:///System.ServiceModel.Configuration.ProtocolMappingSection?qualifyHint=False&amp;autoUpgrade=True)   
- [System.ServiceModel.Configuration.ProtocolMappingElement](assetId:///System.ServiceModel.Configuration.ProtocolMappingElement?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ <xref:System.ServiceModel.Configuration.ProtocolMappingSection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/protocolmapping.md]: "?displayProperty=fullName" for both

File: docs/framework/configure-apps/file-schema/wcf/routing.md [hunk 21]
```diff
@@ -51,4 +51,4 @@ Represents a configuration section for defining a set of routing filters, which
 |system.ServiceModel|The root element of all WCF configuration elements.|  
   
 ## See Also  
- [System.ServiceModel.Routing.Configuration.RoutingSection](assetId:///System.ServiceModel.Routing.Configuration.RoutingSection?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ <xref:System.ServiceModel.Routing.Configuration.RoutingSection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/routing.md]: "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/servicedebug.md [hunk 22]
```diff
@@ -44,11 +44,11 @@ Specifies debugging and help information features for a [!INCLUDE[indigo1](../..
...
 |---------------|-----------------|  
-|httpHelpPageBinding|A string value that specifies the type of binding to be used when HTTP is utilized to access the service help page.<br /><br /> Only bindings with inner binding elements that support [System.ServiceModel.Channels.IReplyChannel](assetId:///System.ServiceModel.Channels.IReplyChannel?qualifyHint=False&amp;autoUpgrade=True) will be supported. Additionally, the [System.ServiceModel.Channels.MessageVersion](assetId:///System.ServiceModel.Channels.MessageVersion?qualifyHint=False&amp;autoUpgrade=True) property of the binding must be [System.ServiceModel.Channels.MessageVersion.None](assetId:///System.ServiceModel.Channels.MessageVersion.None?qualifyHint=False&amp;autoUpgrade=True).|  
+|httpHelpPageBinding|A string value that specifies the type of binding to be used when HTTP is utilized to access the service help page.<br /><br /> Only bindings with inner binding elements that support <xref:System.ServiceModel.Channels.IReplyChannel> will be supported. Additionally, the <xref:System.ServiceModel.Channels.MessageVersion> property of the binding must be <xref:System.ServiceModel.Channels.MessageVersion.None>.|
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/servicedebug.md]: "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/servicedebug.md [hunk 23]
```diff
@@ -44,11 +44,11 @@ Specifies debugging and help information features for a [!INCLUDE[indigo1](../..
...
-|httpsHelpPageBinding|A string value that specifies the type of binding to be used when HTTPS is utilized to access the service help page.<br /><br /> Only bindings with inner binding elements that support assetId:///System.ServiceModel.Channels.IReplyChannel?qualifyHint=False&amp;autoUpgrade=True will be supported. Additionally, the assetId:///System.ServiceModel.Channels.MessageVersion?qualifyHint=False&amp;autoUpgrade=True property of the binding must be assetId:///System.ServiceModel.Channels.MessageVersion.None?qualifyHint=False&amp;autoUpgrade=True.|  
+|httpsHelpPageBinding|A string value that specifies the type of binding to be used when HTTPS is utilized to access the service help page.<br /><br /> Only bindings with inner binding elements that support <xref:System.ServiceModel.Channels.IReplyChannel> will be supported. Additionally, the <xref:System.ServiceModel.Channels.MessageVersion> property of the binding must be <xref:System.ServiceModel.Channels.MessageVersion.None>.|
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/servicedebug.md]: The last two xrefs should be  "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/servicedebug.md [hunk 24]
```diff
@@ -73,7 +73,7 @@ Specifies debugging and help information features for a [!INCLUDE[indigo1](../..
...
+ The optional `httpHelpPageBinding` and `httpHelpPageBinding` attributes allow you to configure the bindings used to access the service web page. If they are not specified, the default bindings (`HttpTransportBindingElement`, in the case of HTTP and `HttpsTransportBindingElement`, in the case of HTTPS) are used for service help page access as appropriate. Notice that you cannot use these attributes with the built-in WCF bindings. Only bindings with inner binding elements that support xref:System.ServiceModel.Channels.IReplyChannel> will be supported. Additionally, the <xref:System.ServiceModel.Channels.MessageVersion> property of the binding must be <xref:System.ServiceModel.Channels.MessageVersion.None>.
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/servicedebug.md]: The last two xrefs should include  "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/state-of-wcf-workflowinstancequery.md [hunk 25]
```diff
@@ -90,8 +90,8 @@ Represents a collection of subscribed states from the tracked workflow instance
 ```  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElement](assetId:///System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElement?qualifyHint=False&amp;autoUpgrade=True)   
- [System.ServiceModel.Activities.Tracking.Configuration.StateElement](assetId:///System.ServiceModel.Activities.Tracking.Configuration.StateElement?qualifyHint=False&amp;autoUpgrade=True)   
- [System.Activities.Tracking.WorkflowInstanceQuery](assetId:///System.Activities.Tracking.WorkflowInstanceQuery?qualifyHint=False&amp;autoUpgrade=True)   
+ <xref:System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElement>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/state-of-wcf-workflowinstancequery.md]: All three should include "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/states-of-wcf-workflowinstancequery.md [hunk 26]
```diff
@@ -89,8 +89,8 @@ Represents a collection of subscribed states from the tracked workflow instance
 ```  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElement](assetId:///System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElement?qualifyHint=False&amp;autoUpgrade=True)   
- [System.ServiceModel.Activities.Tracking.Configuration.StateElementCollection](assetId:///System.ServiceModel.Activities.Tracking.Configuration.StateElementCollection?qualifyHint=False&amp;autoUpgrade=True)   
- [System.Activities.Tracking.WorkflowInstanceQuery](assetId:///System.Activities.Tracking.WorkflowInstanceQuery?qualifyHint=False&amp;autoUpgrade=True)   
+ <xref:System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElement>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/states-of-wcf-workflowinstancequery.md]: All three should include "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/tracking-of-wcf.md [hunk 27]
```diff
@@ -52,5 +52,5 @@ Represents a configuration section for defining tracking settings for a workflow
  Tracking provides you with the ability to examine the execution of a workflow. The workflow tracking infrastructure instruments a workflow to emit records reflecting key events during the execution. For example, when a workflow instance starts or completes tracking records are emitted. Tracking can also extract business relevant data associated with the workflow variables. For example, if the workflow represents an order processing system the order id can be extracted along with the tracking record. In general, enabling WF tracking facilitates diagnostics or business analytics over a workflow execution.  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.TrackingSection](assetId:///System.ServiceModel.Activities.Tracking.Configuration.TrackingSection?qualifyHint=False&amp;autoUpgrade=True)
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/tracking-of-wcf.md]: add "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/workflowinstancequeries-of-wcf.md [hunk 28]
```diff
@@ -79,7 +79,7 @@ Represents a collection of configuration elements that track workflow instance l
 ```  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElementCollection](assetId:///System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElementCollection?qualifyHint=False&amp;autoUpgrade=True)   
- [System.Activities.Tracking.WorkflowInstanceQuery](assetId:///System.Activities.Tracking.WorkflowInstanceQuery?qualifyHint=False&amp;autoUpgrade=True)   
+ <xref:System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElementCollection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/workflowinstancequeries-of-wcf.md]: add "?displayProperty=fullName"  to both xrefs

File: docs/framework/configure-apps/file-schema/wcf/workflowinstancequeries-of-wcf.md [hunk 29]
```diff
@@ -79,7 +79,7 @@ Represents a collection of configuration elements that track workflow instance l
 ```  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElementCollection](assetId:///System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElementCollection?qualifyHint=False&amp;autoUpgrade=True)   
- [System.Activities.Tracking.WorkflowInstanceQuery](assetId:///System.Activities.Tracking.WorkflowInstanceQuery?qualifyHint=False&amp;autoUpgrade=True)   
+ <xref:System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElementCollection>    
+ <xref:System.Activities.Tracking.WorkflowInstanceQuery>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/workflowinstancequeries-of-wcf.md]: "?displayProperty=fullName"

File: docs/framework/configure-apps/file-schema/wcf/workflowinstancequery-of-wcf.md [hunk 30]
```diff
@@ -80,7 +80,7 @@ Represents a query that tracks workflow instance life cycle changes such as a st
 ```  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElement](assetId:///System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElement?qualifyHint=False&amp;autoUpgrade=True)   
- [System.Activities.Tracking.WorkflowInstanceQuery](assetId:///System.Activities.Tracking.WorkflowInstanceQuery?qualifyHint=False&amp;autoUpgrade=True)   
+ <xref:System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElement>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/workflowinstancequery-of-wcf.md]: "?displayProperty=fullName"  for both

File: docs/framework/configure-apps/file-schema/wcf/workflowinstancequery-of-wcf.md [hunk 31]
```diff
@@ -80,7 +80,7 @@ Represents a query that tracks workflow instance life cycle changes such as a st
 ```  
   
 ## See Also  
- [System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElement](assetId:///System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElement?qualifyHint=False&amp;autoUpgrade=True)   
- [System.Activities.Tracking.WorkflowInstanceQuery](assetId:///System.Activities.Tracking.WorkflowInstanceQuery?qualifyHint=False&amp;autoUpgrade=True)   
+ <xref:System.ServiceModel.Activities.Tracking.Configuration.WorkflowInstanceQueryElement>    
+ <xref:System.Activities.Tracking.WorkflowInstanceQuery>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/workflowinstancequery-of-wcf.md]: "?displayProperty=fullName"

**Review by rpetrusha — CHANGES_REQUESTED** [review]

**Comment by rpetrusha** [comment]
I went through about half of these in detail, but where the fully qualified name appears in brackets, such as [System.ServiceModel.SomeType](<assetId:T:System.ServiceModel.Routing.Configuration.RoutingSection>), this should be translated either as [System.ServiceModel.SomeType](<xref:System.ServiceModel.Routing.Configuration.RoutingSection>) or, preferably, as <xref:System.ServiceModel.Routing.Configuration.RoutingSection?displayProperty=fullName>.

A second issue, which I've noticed in a number of the topics, is that when there are multiple See Also items, there must be three spaces at the end of a line that is followed by another item. Otherwise, they'll be rendered on a single line.

**Comment by tompratt-AQ** [comment]
@rpetrusha  I went through all the changes and made corrections as you suggested. All of my fixes, except for two, are in See Also links where I completely missed adding the display=fullname and adding 3 spaces at the end of the line in some cases.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @tompratt-AQ. This is ready to merge.

**Comment by GuardRex** [comment]
@rpetrusha

> A second issue, which I've noticed in a number of the topics, is that when there are multiple See Also items, there must be three spaces at the end of a line that is followed by another item. Otherwise, they'll be rendered on a single line.

Just learning this. Over on the ASP.NET Core side, every "See also/Additional resources" list is a bullet list, so it wasn't an issue. I just spoke with @mairaw, and I'll go with 3-spaces, no blank lines between resources, and no bullets over here. I'm running through all of my open PR's now to correct this.

**Comment by rpetrusha** [comment]
@GuardRex, sorry that I didn't notice that -- it's something I always forget about until I eventually see topics published live.

**Comment by GuardRex** [comment]
@rpetrusha It was ok ... only one doc on one PR needed a fix ... it was the one we're just about to merge: The wcf migration links update. I **definitely** want to go back and make a pass over merged PR's to see if I ... er ... "fixed" :grinning: ... any other "See also" lists. I'll get 'um patched in a new PR if I find any.

**Pull request merged**